packages/forms-web-app/src/dynamic-forms/question.js

```diff
@@ -63,6 +63,8 @@ class Question {
 	html;
 	/** @type {string|undefined} optional question type */
 	type;
+	/** @type {(response: JourneyResponse) => boolean} */
+	shouldDisplay = () => true;
```

Comment:
minor point but shouldDisplay feels little frontend/display focussed, can't currently think of a good enough alternative name to suggest though so happy to keep

Reply:
That is a fair point. Happy to rename in future if we think of something better